vars/runSmokeTestScript.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+/**@
+ * Contains logic to execute the smoke test workflow
+ *
+ * @param args A map of the following parameters
+ * @param args.testManifest <required> Test manifest file location
+ * @param args.buildManifest <required> Build manifest file location
+ * @param args.jobName <optional> Job name that triggered the workflow. 'distribution-build-opensearh' by default.
```

Comment:
Nit: make optional param at the end of the list.

Reply:
Updated. Thanks!